x-pack/plugins/security_solution/public/management/components/administration_list_page.tsx

```diff
@@ -63,24 +63,18 @@ export const AdministrationListPage: FC<AdministrationListPageProps & CommonProp
               name: ENDPOINTS_TAB,
               id: AdministrationSubTab.endpoints,
               href: getEndpointListPath({ name: 'endpointList' }),
-              urlKey: 'administration',
-              pageId: SecurityPageName.administration,
```

Comment:
These `urlKey` and `pageId` not needed anymore? just to know

Reply:
Now that `SecuritySolutionTabNavigation` is only used for SubPlugins internal tabs (Network, Hosts, and Administration), I could delete the logic for handling the navigation between SubPlugins. It includes deleting `urlKey` and `pageId`.

* `urlKey` was used for building a new query string for a SubPlugin. since all tabs are inside a subPlugin, the query string is always the same.
* `pageId` was used to determine if the tab link should navigate to a new SubPlugin or an internal tab. Now it always navigates to an internal tab with `history.push``.